documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,574 @@
+  describing the problem the proposal aims to solve. In the original Sociocratic
+  model this is called the "driver".
+
+## Process summary
+
+The process breaks proposal evaluation and decision-making into a series of
+rounds. Each round has expectations about what kinds of feedback are
+appropriate. These steps are ordered and must be followed in the order they are
+presented here.
+
+Full descriptions for each of these rounds are below. This section is a summary
+meant to give an overview for people new to or refreshing themselves on the
+process.
+
+> **Note** The "goal" column is a very brief summary and is not intended to
+> encapsulate the full scope of the round. Please refer to the
+> [round descriptions below](#round-descriptions) for more details and examples.
+
+| round                                                 | Suggested span      | Goal                                                                                      |
+| ----------------------------------------------------- | ------------------- | ----------------------------------------------------------------------------------------- |
+| [Question round](#question-round)                     | 3 days              | Clarify the proposal; share initial reactions                                             |
+| [Revision round](#revision-round)                     | Author's discretion | Update the text of the proposal to reflect the outcome of the previous round              |
+| [Objection round](#objection-round)                   | 2 days              | Identify paramount and non-paramount objections                                           |
+| [Objection revision round](#objection-revision-round) | Author's discretion | Work with participants to revise the text of the proposal to address paramount objections |
+| [Approval](#approval)                                 | N/A                 | Mark as approved and create issues to implement the proposal                              |
+| [Tabling](#tabling)                                   | N/A                 | Indicate that a proposal will not be implemented                                          |
+
+## Proposal requirements
+
+Project proposals and technical implementation plans should follow the formats
+outlined in [the documentation for those processes](./docs/projects/README.md).
+All proposals should include a summary of the purpose of the proposal and the
+problem(s) it attempts to address.
+
+Every proposal must include the following boilerplate text in the PR description
+or at the top of the GitHub Discussion:
+
+```md
+## Decision-making process
+
+For the purposes of this discussion, we will follow
+[Openverse's decision-making model](https://wordpress.github.io/openverse/reference/consent_decision_making.html).
+This process follows formalised steps with specific expectations of
+participants. Before contributing, please read
```

Comment:
I am trying to word my suggestion as a question:)

Could we not make it sound as if following the process is a hard requirement? 

It seems a very large ask for the community to read and understand a long document before adding a short comment. For instance, we might want some feedback from the designers or want ideas from the larger community on devex. I think having the requirements written in such a way might push non-maintainers away from contributing.

Having a clear indication of the current stage and the links to project documentation is very useful for maintainers. Reminding to read the docs shouldn't be necessary for the maintainers and might be off-putting for outside contributors.

Reply:
For sure. Zack brought a similar concern up that it's a long document to ask someone who is making an occasional consultation-style contribution to a discussion to read and understand.

I'd like to add something to qualify that, basically to carve out an exception for folks to be able to participate in that way when necessary. I do think it's useful to have the expectation that the process will be followed, but I also think we're capable of managing it on our own and deciding whether we're getting sidetracked from the core discussion.